src/plugins/ssr.js

```diff
@@ -17,9 +16,7 @@ import type {
 const SSRDecider = createPlugin({
   provides: () => {
     return ctx => {
-      // If the request has one of these extensions, we assume it's not something that requires server-side rendering of virtual dom
-      // TODO(#46): this check should probably look at the asset manifest to ensure asset 404s are handled correctly
-      if (ctx.path.match(/\.(js|gif|jpg|png|pdf|json)$/)) return false;
+      if (ctx.path.startsWith(ctx.assetBase)) return false;
       // The Accept header is a good proxy for whether SSR should happen
```

Comment:
We should use `req.accepts(['html', 'json']) === 'html'` instead ?

Reply:
Should we also provide `whitelist extensions` as default for user to override/extend ?